.gitlab/pkg_metrics.yml

```diff
@@ -303,6 +303,8 @@ check_pkg_size-amd64-a7:
     - agent_suse-x64-a7
     - dogstatsd_suse-x64
     - iot_agent_suse-x64
+    - agentless_scanner_deb-x64
+    - agentless_scanner_rpm-x64
```

Comment:
I suppose we could also monitor the suse package here

Reply:
Right. I've added the SUSE package as well.